doc/developer-guides/hld/hld-APL_GVT-g.rst

```diff
@@ -116,9 +116,9 @@ API Forwarding
 
 API forwarding, or a split driver model, is another widely-used I/O
 virtualization technology. It has been used in commercial virtualization
-productions, for example, VMware*, PCoIP*, and Microsoft* RemoteFx*.
+productions such as VMware*, PCoIP*, and Microsoft* RemoteFx*.
 It is a natural path when researchers study a new type of
-I/O virtualization usage, for example, when GPGPU computing in VM was
+I/O virtualization usage—for example, when GPGPU computing in a VM was
```

Comment:
Is this a hyphen in between `usage` and `for`?

Reply:
Those are em dashes, and when rendered they'll be nice and wide (not like hyphens).

Reply:
Thanks for the clarification 👍